src/bitcoind.cpp

```diff
@@ -152,6 +151,7 @@ static bool AppInit(int argc, char* argv[])
             // If locking the data directory failed, exit immediately
             return false;
         }
+        AppInitInterfaces(node);
```

Comment:
Why ignore returned value (even if only `true` at this point)?

```suggestion
        if (!AppInitInterfaces(node)) return false;
```

Reply:
seems to be addressed